regulatory/data/history.yml

```diff
@@ -0,0 +1,35 @@
+change_requests:
+- id: '1'
+  title: Feature Change Request
+  content: A description of the requested changes should go here.  Usually,
+    this data is periodically pulled out of a separate project management tool,
+    such as github
+  is_problem_report: false
+  change_ids:
+  - '1'
+- id: '2'
+  title: Problem Report
+  content: A problem report may or may not also be a change request.  If the
+    reported problem doesn't require changes, then it is just a "known anomaly"
+    that won't be addressed.  If it does require changes, it also acts as a
+    change request.
+  is_problem_report: true
+  change_ids: []
+changes:
+- id: '1'
```

Comment:
...is this a dummy example, or is it something real?

Reply:
(Same elsewhere)

Reply:
This is a dummy file and automatically generated when installing rdm and should have information manually entered. It creates data in the release/release_history.md file

Reply:
This file is intended to be generated via a script pulling out of GitHub Issues or pull request.